games-arcade/caveexpress/patches/caveexpress-2.4.patchset

```diff
@@ -0,0 +1,228 @@
++		if (TEXTURE_PACKER_EXPORTER MATCHES "Unknown format")
++			message(STATUS "Found TexturePacker: ${TEXTUREPACKER_BIN}")
++			message(WARNING "but without the needed exporter. Find it in contrib/assets/png-packed/exporter. Check out the HOWTO.md in that directory.")
++			unset(TEXTUREPACKER_BIN)
++		else()
++			message(STATUS "Found TexturePacker: ${TEXTUREPACKER_BIN}")
++		endif()
+ 	else()
+ 		message(STATUS "Could not find TexturePacker")
+ 	endif()
+@@ -112,6 +133,8 @@ if (NOT CMAKE_TOOLCHAIN_FILE)
+ 		include(${CP_CMAKE_DIR}/toolchains/windows-toolchain.cmake)
+ 	elseif (DARWIN)
+ 		include(${CP_CMAKE_DIR}/toolchains/darwin-toolchain.cmake)
++	elseif (HAIKU)
++		include(${CP_CMAKE_DIR}/toolchains/haiku-toolchain.cmake)
+ 	else()
+ 		include(${CP_CMAKE_DIR}/toolchains/linux-toolchain.cmake)
+ 	endif()
+@@ -129,13 +152,14 @@ elseif (NACL)
+ 	include(${CP_CMAKE_DIR}/nacl.cmake)
+ elseif (EMSCRIPTEN)
+ 	include(${CP_CMAKE_DIR}/emscripten.cmake)
++elseif (HAIKU)
++#	include(${CP_CMAKE_DIR}/haiku.cmake)
+ endif()
+ 
+ if (PKGDATADIR)
+ 	add_definitions(-DPKGDATADIR="${PKGDATADIR}")
+-	message(STATUS "You specified a package data dir - but this means, you have to rebuild for each game.")
+-	message(STATUS "If you don't do this, you get all the data of all the games in the same directory.")
+-	message(STATUS "See the available options and disable all but one game for each build.")
++	message(STATUS "You specified a package data dir - the game will append the game name to this directory.")
++	message(STATUS "So you get e.g. ${PKGDATADIR}/caveexpress as final directory.")
+ endif()
+ 
+ if (NOT NETWORKING)
+@@ -167,12 +191,23 @@ include_directories(${CMAKE_BINARY_DIR})
+ 
+ include_directories(src)
+ include_directories(src/modules)
+-if (UNITTESTS)
+-	include_directories(src/libs/gtest)
+-	include_directories(src/libs/gtest/include)
+-endif()
++# Packagers might want to remove this
++if (EXISTS "${ROOT_DIR}/src/libs/" AND NOT FORCE_USE_SYSTEM_LIBS)
++	if (UNITTESTS)
++		include_directories(src/libs/gtest)
++		include_directories(src/libs/gtest/include)
++	endif()
+ 
+-add_subdirectory(src/libs)
++	add_subdirectory(src/libs)
++else()
++	include(FindPkgConfig)
++	PKG_SEARCH_MODULE(SDL2 REQUIRED sdl2>=2.0.3)
++	PKG_SEARCH_MODULE(SDL2IMAGE REQUIRED SDL2_image>=2.0.0)
++	PKG_SEARCH_MODULE(LUA5.2 REQUIRED lua5.2)
++	PKG_SEARCH_MODULE(BOX2D REQUIRED box2d>=2.3.0)
++	PKG_SEARCH_MODULE(SQLITE3 REQUIRED sqlite3)
++	include_directories(${SDL2_INCLUDE_DIRS} ${SDL2IMAGE_INCLUDE_DIRS})
++endif()
+ 
+ # TODO move into lib cmakefiles
+ if (LUA5.2_FOUND)
+@@ -193,4 +228,4 @@ if (MINIRACER)
+ endif()
+ if (UNITTESTS)
+ 	add_subdirectory(src/tests)
+-endif()
++endif()
+\ No newline at end of file
+diff --git a/cmake/toolchains/haiku-toolchain.cmake b/cmake/toolchains/haiku-toolchain.cmake
+new file mode 100644
+index 0000000..7045206
+--- /dev/null
++++ b/cmake/toolchains/haiku-toolchain.cmake
```

Comment:
Looks like this is a copy/paste from a merged PR by @extrowerk ?
https://github.com/mgerhardy/caveexpress/blob/a96cccb9ae5442514471e4901351b679470fcae1/cmake/toolchains/haiku-toolchain.cmake

Reply:
Exactly, this is it. I expected these files will download without srcGitRev but this not happend so I created patchset with these files.